ts/Series/BoxPlot/BoxPlotSeries.ts

```diff
@@ -87,6 +87,65 @@ class BoxPlotSeries extends ColumnSeries {
         return {};
     }
 
+
+    // Get an SVGPath object for both whiskers
+    public getWhiskerPair(
+        halfWidth: number,
+        stemX: number,
+        upperWhiskerLength: number | string,
+        lowerWhiskerLength: number | string,
+        point: BoxPlotPoint
+    ): SVGPath {
+        const strokeWidth = point.whiskers.strokeWidth(),
+            getWhisker = (
+                xLen: (
+                    number |
+                    string |
+                    undefined
+                ),
+                yPos: number
+            ): SVGPath.Segment[] => {
+                const halfLen = (
+                        (
+                            typeof xLen === 'string' &&
+                            xLen.endsWith('%')
+                        ) ?
+                            (
+                                halfWidth * parseFloat(xLen)
+                            ) / 100 :
+                            Number(xLen) / 2
```

Comment:
We probably want to use the `relativeLength` utility function that is designed exactly for this pattern.